tests/SecurityDetails/SecurityDetailsConnector.test.ts

```diff
@@ -49,4 +49,90 @@ export async function securityDetailsLoad (
         }
     )
 
+    const AssetAllocationsConnector = new MC.SecurityDetailsConnector({
+        api,
+        security: {
+            id: 'F0GBR050DD',
+            idType: 'MSID'
+        }
```

Comment:
No need to specify `type`? Here in all other tests? Why the tests pass without it?